lib/game_looper.rb

```diff
@@ -17,34 +15,30 @@ def loop
 
   private
 
-  attr_reader :console, :display, :prompt, :board, :players, :outcome_checker
+  attr_reader :ui, :board, :players, :outcome_checker
 
   def game_is_over
     outcome_checker.draw? || outcome_checker.win?
   end
 
   def take_turns(players)
     players.each do |player|
+      display_board
       take_turn(player) unless game_is_over
     end
   end
 
   def take_turn(player)
-    display_board
     move = get_move
     mark_board(player, move)
   end
 
   def display_board
-    board_display = display.present(board)
-    console.output(board_display)
+    ui.display_board(board)
   end
 
   def get_move
-    message = 'Enter a number 1-9: '
-    error_message = 'Please enter a valid number.'
-
-    prompt.call(message, error_message)
+    ui.prompt
```

Comment:
Good thoughts! I ended up moving get_move to UI which cleans things up, and I like the naming of UI#get_move instead of UI#prompt. I didn't end up using the delegator in this case because I wanted to stay consistent with how other methods are being called in Game Looper. I'll definitely keep an eye out for another situation to use delegation though!